tox.ini

```diff
@@ -1,15 +1,12 @@
 [tox]
-envlist = py27, py33, py34, py35, py36, pypy, lint
+envlist = py27, py34, py35, py36, py37, lint
 
 [testenv]
-deps= -r{toxinidir}/test_requirements.txt
-commands= py.test -n 4 --cov hyperframe {toxinidir}/test/
-
-[testenv:pypy]
-# temporarily disable coverage testing on PyPy due to performance problems
-commands= py.test -n 4 hyperframe {toxinidir}/test/
+deps = -r{toxinidir}/test_requirements.txt
+commands =
+    coverage run -m pytest --cov=hyperframe {posargs}
```

Comment:
Does this `--cov` flag get absorbed by `.coveragerc`?

Reply:
No - we still need to tell pytest which module it should collect coverage data on.